src/ensembl/src/shared/accordion/components/AccordionItem.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Accordion from './Accordion';
+import AccordionItem from './AccordionItem';
+
+describe('AccordionItem', () => {
+  it('renders without erroring', () => {
+    expect(() => {
+      mount(<Accordion />);
+    }).not.toThrow();
+  });
+
+  describe('className prop', () => {
+    it('is “accordionItem by default', () => {
+      const wrapper = mount(
+        <Accordion>
+          <AccordionItem uuid={'FOO'} />
+        </Accordion>
+      );
+      expect(wrapper.find(AccordionItem).props()['className']).toEqual(
+        'accordionItem'
+      );
+    });
+
+    it('can be overridden', () => {
+      const wrapper = mount(
+        <Accordion>
+          <AccordionItem uuid={'FOO'} className="foo" />
+        </Accordion>
+      );
+      expect(wrapper.find(AccordionItem).props()['className']).toEqual('foo');
+    });
```

Comment:
@imran-ebi, here's where we had a difference of opinion.

One way of styling accordion components is, like in the test, by replacing default class with a provided class. The `accordionItem` class gets replaced by `foo`.

Another way of styling would be to extend the default class with a custom class. The result would be `accordionItem foo`.

An argument against extending is that if the default class gets updated, it will (or might) cause changes in all extended classes. I.e. if you take accordion and put it in a specific context overwriting some of its styles, then after update to the default class, the accordion in that specific context will also get updated (which, I would argue, is something that we might actually want).

An argument for extending is that wherever you do want to apply custom classes, you do not have to define all style rules, but can only update the relevant ones.

Which would you prefer?

Reply:
As we discussed yesterday, I would prefer extending the styling. Having said that, we can change the implementation of this to overriding the default styling if the accordions don't look similar to each other.